lib/hanami/generators/app/application.rb.tt

```diff
@@ -205,6 +205,12 @@ module <%= config[:classified_app_name] %>
       # Inline JavaScript is NOT allowed. To enable it, please use:
       # "script-src 'unsafe-inline'".
       #
+      # Javascript eval() and similar methods are NOT allowed. It needs to be enabled
+      # for Javascript frameworks that rely on
+      # such features, such as vue.js. To enable it, please use:
+      # "script-src unsafe-eval'".
```

Comment:
I don't know much about how JS MVC frameworks work in the browser. But there's `unsafe-inline` and `unsave-eval`. 

Vue needs the latter (according to @mytram), but I'm not sure about the others :)